### PR TITLE
Update limits page

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Generalize Usage limits page to apply to both Cloud and Enterprise, update old links.

### CODE REVIEW

**Review by jason-seqera — APPROVED** [review]
LGTM

File: platform_versioned_docs/version-23.3.0/limits/limits.mdx [hunk 1]
```diff
@@ -5,13 +5,13 @@ date: "21 Apr 2023"
...
 
-[Seqera Cloud](https://tower.nf) elements and features have default limits per workspace and organization.
+Seqera Platform elements and features have default limits per workspace and organization.
 
 ### Workspaces
 
 | Description  | Value |
 | ------------ | ----- |
-| Active runs  | 5     |
+| Active runs  | Cloud: 5 Enterprise: Unlimited    |
```
Comment (tcrespog) [inline platform_versioned_docs/version-23.3.0/limits/limits.mdx:14]: Sorry, @llewellyn-sl, I was wrong. I've re-checked and the default limit through License Manager is [5](https://github.com/seqeralabs/license-man/blob/26d503f6e34f78d22d9f366a881425a14ccca7b3/lm-backend/src/main/resources/db/migration/V04__add_default_quota_values.sql#L44-L45) too. Through config file, it is unlimited.

Comment (llewellyn-sl) [reply platform_versioned_docs/version-23.3.0/limits/limits.mdx]: @tcrespog Thanks. I've added an admonition to the page and added that config variable to the install guide.

File: platform_versioned_docs/version-23.4.0/limits/limits.mdx [hunk 2]
```diff
@@ -18,6 +18,12 @@ tags: [limits]
 | Datasets     | 100   |
 | Labels       | 1000  |
 
+:::note 
+Some Enterprise instances on older licenses are limited to 100 labels per workspace. [Contact support](mailto:support@seqera.io) to upgrade your license.
+
+Enterprise customers can increase the maximum active runs per workspace by setting the `TOWER_LAUNCH_MAXRUNS` [environment variable](../enterprise/configuration/overview.mdx#basic-configuration).
```
Comment (tcrespog) [inline platform_versioned_docs/version-23.4.0/limits/limits.mdx:24]: Similarly to other limits, the "max runs" limit is also configured through the License Manager. Setting `TOWER_LAUNCH_MAXRUNS` may not have any effect for those.

Comment (llewellyn-sl) [reply platform_versioned_docs/version-23.4.0/limits/limits.mdx:24]: Thanks meneer. I'll remove the max runs piece then.

**Review by tcrespog — APPROVED** [review]

**Pull request merged**